run/shared/settings/geosfp/geosfp.preprocessed_ll.txt

```diff
@@ -3,7 +3,7 @@ RUNDIR_MET_LOWERCASE='geosfp'
 RUNDIR_MET_DIR="${RUNDIR_DATA_ROOT}/GEOS_${RUNDIR_GRID_DIR}/GEOS_FP"
 RUNDIR_MET_DIR_RAW="${RUNDIR_DATA_ROOT}/GEOS_0.25x0.3125/GEOS_FP"
 
-RUNDIR_METEOROLOGY_VERTICAL_INDEX_IS_TOP_DOWN=.false.
+RUNDIR_MET_OTHER_IS_TOP_DOWN=.false.
```

Comment:
@lizziel: Should this be `MET_NONADVECTION_IS_TOP_DOWN=.false.` here?

Reply:
Yeah, I was going back and forth on what to name this. I'll change it to specify non-advection rather than other.